# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/use_counter_metrics.yaml
# Commit: ebbd2cd3905e
# Full Hash: ebbd2cd3905e3542c417ea33e0c2cdd88260e6c9
# Author: Cristina Horotan <chorotan@mozilla.com>
# Date: 2025-05-22 09:08:48
# Regressor Bug: 1964451
# File Overlap Count: 1
# Description:
#   Revert "Bug 1964451 - Add use counters for OOM handling r=sfink,mccr8" on request
#   
#   This reverts commit 7ae56a029252266ce967356e2254b1408b8522ea.
# ==============================================================================

diff -r 34bb3734b886 -r ebbd2cd3905e dom/base/use_counter_metrics.yaml
--- a/dom/base/use_counter_metrics.yaml	Wed May 21 18:52:10 2025 +0000
+++ b/dom/base/use_counter_metrics.yaml	Wed May 21 22:38:12 2025 +0300
@@ -107,8 +107,8 @@
     send_in_pings:
       - use-counters
 
-# Total of 2405 use counter metrics (excludes denominators).
-# Total of 362 'page' use counters.
+# Total of 2397 use counter metrics (excludes denominators).
+# Total of 358 'page' use counters.
 use.counter.page:
   svgsvgelement_getelementbyid:
     type: counter
@@ -824,74 +824,6 @@
     send_in_pings:
       - use-counters
 
-  js_large_oom_reported:
-    type: counter
-    description: >
-      Whether a page reported a large oom.
-      Compare against `use.counter.top_level_content_documents_destroyed`
-      to calculate the rate.
-    bugs:
-      - https://bugzilla.mozilla.org/show_bug.cgi?id=1852098
-    data_reviews:
-      - https://bugzilla.mozilla.org/show_bug.cgi?id=1852098
-    notification_emails:
-      - dom-core@mozilla.com
-      - emilio@mozilla.com
-    expires: never
-    send_in_pings:
-      - use-counters
-
-  js_small_oom_reported:
-    type: counter
-    description: >
-      Whether a page reported a small oom.
-      Compare against `use.counter.top_level_content_documents_destroyed`
-      to calculate the rate.
-    bugs:
-      - https://bugzilla.mozilla.org/show_bug.cgi?id=1852098
-    data_reviews:
-      - https://bugzilla.mozilla.org/show_bug.cgi?id=1852098
-    notification_emails:
-      - dom-core@mozilla.com
-      - emilio@mozilla.com
-    expires: never
-    send_in_pings:
-      - use-counters
-
-  js_large_oom_recovered:
-    type: counter
-    description: >
-      Whether a page recovered a large oom.
-      Compare against `use.counter.top_level_content_documents_destroyed`
-      to calculate the rate.
-    bugs:
-      - https://bugzilla.mozilla.org/show_bug.cgi?id=1852098
-    data_reviews:
-      - https://bugzilla.mozilla.org/show_bug.cgi?id=1852098
-    notification_emails:
-      - dom-core@mozilla.com
-      - emilio@mozilla.com
-    expires: never
-    send_in_pings:
-      - use-counters
-
-  js_small_oom_recovered:
-    type: counter
-    description: >
-      Whether a page recovered a small oom.
-      Compare against `use.counter.top_level_content_documents_destroyed`
-      to calculate the rate.
-    bugs:
-      - https://bugzilla.mozilla.org/show_bug.cgi?id=1852098
-    data_reviews:
-      - https://bugzilla.mozilla.org/show_bug.cgi?id=1852098
-    notification_emails:
-      - dom-core@mozilla.com
-      - emilio@mozilla.com
-    expires: never
-    send_in_pings:
-      - use-counters
-
   console_assert:
     type: counter
     description: >
@@ -6264,7 +6196,7 @@
     send_in_pings:
       - use-counters
 
-# Total of 362 'document' use counters.
+# Total of 358 'document' use counters.
 use.counter.doc:
   svgsvgelement_getelementbyid:
     type: counter
@@ -6980,74 +6912,6 @@
     send_in_pings:
       - use-counters
 
-  js_large_oom_reported:
-    type: counter
-    description: >
-      Whether a document reported a large oom.
-      Compare against `use.counter.content_documents_destroyed`
-      to calculate the rate.
-    bugs:
-      - https://bugzilla.mozilla.org/show_bug.cgi?id=1852098
-    data_reviews:
-      - https://bugzilla.mozilla.org/show_bug.cgi?id=1852098
-    notification_emails:
-      - dom-core@mozilla.com
-      - emilio@mozilla.com
-    expires: never
-    send_in_pings:
-      - use-counters
-
-  js_small_oom_reported:
-    type: counter
-    description: >
-      Whether a document reported a small oom.
-      Compare against `use.counter.content_documents_destroyed`
-      to calculate the rate.
-    bugs:
-      - https://bugzilla.mozilla.org/show_bug.cgi?id=1852098
-    data_reviews:
-      - https://bugzilla.mozilla.org/show_bug.cgi?id=1852098
-    notification_emails:
-      - dom-core@mozilla.com
-      - emilio@mozilla.com
-    expires: never
-    send_in_pings:
-      - use-counters
-
-  js_large_oom_recovered:
-    type: counter
-    description: >
-      Whether a document recovered a large oom.
-      Compare against `use.counter.content_documents_destroyed`
-      to calculate the rate.
-    bugs:
-      - https://bugzilla.mozilla.org/show_bug.cgi?id=1852098
-    data_reviews:
-      - https://bugzilla.mozilla.org/show_bug.cgi?id=1852098
-    notification_emails:
-      - dom-core@mozilla.com
-      - emilio@mozilla.com
-    expires: never
-    send_in_pings:
-      - use-counters
-
-  js_small_oom_recovered:
-    type: counter
-    description: >
-      Whether a document recovered a small oom.
-      Compare against `use.counter.content_documents_destroyed`
-      to calculate the rate.
-    bugs:
-      - https://bugzilla.mozilla.org/show_bug.cgi?id=1852098
-    data_reviews:
-      - https://bugzilla.mozilla.org/show_bug.cgi?id=1852098
-    notification_emails:
-      - dom-core@mozilla.com
-      - emilio@mozilla.com
-    expires: never
-    send_in_pings:
-      - use-counters
-
   console_assert:
     type: counter
     description: >